lib/Wikia/src/Logger/LogstashFormatter.php

```diff
@@ -20,6 +20,7 @@ public function __construct()
 
 	protected function formatV0(array $record) {
 		$message = array(
+			'appname' => 'mediawiki',
```

Comment:
`@fields.app_name` could probably be removed soon, `program` is a core thing and setting it up to a specific value shouldn't do any harm, makes sense?